src/usr/local/www/system_usermanager.php

```diff
@@ -893,6 +896,15 @@ function moveOptions(From, To)	{
 			$('form').submit();
 		}
 	});
+	$('[id^=delprivid]').click(function(event) {
+		if(confirm(event.target.title)) {
+			$('#privid).val(event.target.id.match(/\d+$/)[0]);
+			$('#userid').val('<?=$id;?>');
```

Comment:
Missing a closing ' here I think. Should be:

$('#privid').val(event.target.id.match(/\d+$/)[0]);
              ^